docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -67,6 +79,24 @@ If the directory already exists, the generation fails.
 A pair of Dockerfiles for native and jvm mode are also generated in `src/main/docker`.
 Instructions to build the image and run the container are written in those Dockerfiles.
 
+== Using platform
+
+By default the Maven plugin currently uses the extensions that comes with Quarkus core. It is possible to get access to the full ecosystem
```

Comment:
it's only for the create operation, better clarify

Reply:
fixed.